### DISCUSSION TITLE
While Moving/Resizing Action handlig

### DISCUSSION THREAD

**Discussion by Ichisich** [RamonUnch/AltDrag#367] [General]
As it is quite subjective what can feel janky or not, which actions should stop the moving/resize process?

Currently, what I think would profit from continuous movement/resizing:
- Roll/Unroll Window
- Center Window
- Maximize Vertically
- Snap to Side/Corner
- Extend to Side/Corner

The other entries already align with my preference.

**Comment by RamonUnch** [comment]
For now this is a technical limitation, as soon as the action changes the size/position of the selected window, proper handling of combo action would require significant code rework.

**Comment by Ichisich** [comment]
Interesting.

So setting `Maximize window` `While moving` to a dedicated button, that happens to be used to resize, is more limited than the legacy setting:
`Toggle maximize state with the resize button while moving`

**Reply by RamonUnch** [reply]
Yes indeed because there is a ton of specific code just for this one. It is still available if you do not override the resize button.
